### ISSUE TITLE
Scale issues

### ISSUE THREAD

**Issue by Squall789** [JustinPedersen/maya_fspy#9]
I've been using this for a while and its great, works a charm every time, the only issue I run into is scale, the camera is always very close to the center of the grid, is it possible to incorporate some kind of scale functionality so that we can have the camera  be at a larger distance from the get go?

I'm not sure what issues this might present, if any.

**Comment by JustinPedersen96** [comment]
@Squall789 Glad to hear its working for you! 

I could possibly add a rig and an attribute to the camera that would pull it back in the way that you've described?

**Comment by Squall789** [comment]
> @Squall789 Glad to hear its working for you!
> 
> I could possibly add a rig and an attribute to the camera that would pull it back in the way that you've described?

I think that'd work yes, really appreciate the ongoing support here, awesome work.
If you used a rig would that bake the new values at the end? If not, it isn't time consuming for users to bake that cameras position to another camera. I ask because I'm largely exporting to unreal and unreals importer (same for many software I guess) doesn't support the rig, so it wouldn't see those additional offsets, so I always have to bake it out to another camera at the end.

**Comment by JustinPedersen96** [comment]
Oh sweet, I've recently started messing with Unreal too, Loving it so far! 

For the rig side is should be a simple break any connections and then the camera should be positioned where its needed. Would you be interested in testing out anything I come up with?

**Comment by Squall789** [comment]
> Oh sweet, I've recently started messing with Unreal too, Loving it so far!
> 
> For the rig side is should be a simple break any connections and then the camera should be positioned where its needed. Would you be interested in testing out anything I come up with?

I'd love to test as you go.
Since you're also using unreal I'd be interested to discuss something in particular with you about that, do you have an email account or anything I could sent to?

**Comment by JustinPedersen96** [comment]
Yeah you can drop me a mail on justin@tcgcape.co.za and we can chat more there :)
